cfme/fixtures/pytest_selenium.py

```diff
@@ -484,6 +510,8 @@ def handle_alert(cancel=False, wait=30.0, squash=False, prompt=None):
             logger.info("Typing in: {}".format(prompt))
             popup.send_keys(prompt)
         popup.dismiss() if cancel else popup.accept()
+        # Should any problematic "double" alerts appear here, we don't care, just blow'em away.
+        dismiss_any_alerts()
```

Comment:
@psav I'd like to ask you what do you think of this? This might be causing some of the `h is null` errors like what I have fixed in this PR. I added a couple of new functions to support this functionality.